scripts/soltest.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+REPO_ROOT="$(dirname "$0")"/..
+DEBUG_PREFIX=
+BOOST_OPTIONS=
+SOLTEST_OPTIONS=
+CURRENT_OPTIONS=BOOST_OPTIONS
+
+while [ $# -gt 0 ]
+do
+	case "$1" in
+		--debug)
+			shift
+			DEBUG_PREFIX="gdb $1"
```

Comment:
Why `$1`?

Reply:
The arguments are already shifted - the idea was to use

./scripts/soltest.sh --debug "--argument-for-gdb --another-argument-for-gdb" --some-boost-option -- --some-soltest-option

Reply:
in 99.9% of cases, I use `gdb --args soltest ...`